.gitignore

```diff
@@ -50,3 +50,6 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+# IDE - Idea
+.idea
```

Comment:
Do not append your personal settings.

You could use global ignore instead, see [**Configuring ignored files for all repositories on your computer**](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer).